app/src/layouts/common/appSidebar/appSidebar.jsx

```diff
@@ -55,13 +55,14 @@ export const AppSidebar = ({
     setIsHoveredUser(false);
   };
 
-  const createFooterBlock = (openNavbar) => (
+  const createFooterBlock = (openNavbar, onImmediatelyOpenSidebar) => (
     <>
       <div className={cx('policy-block')} />
       <UserAvatar
         onClick={() => {
           openNavbar();
           setIsOpenAvatarPopover(true);
+          onImmediatelyOpenSidebar();
```

Comment:
<Optional> May be better move it to openNavbar and call inside?
openNavbar(onImmediatelyOpenSidebar)

Reply:
These are different functions responsible for different events

Reply:
This behavior was removed